### ISSUE TITLE
Get images of letters and link them to text

### ISSUE THREAD

**Issue by lklic** [villaitatti/yashiro_project#4]
need to get images from the I Tatti digital archive and from the partner institution in Japan

**Comment by gspinaci** [comment]
Should we use the IIIF Viewer?

**Comment by lklic** [comment]
yes, we should use mirador for these. We should also create a manifest for multipage letters.

I have uploaded a few letters to the itatti IIIF server:

https://iiif.itatti.harvard.edu/iiif/2/yashiro!32044138547039_01.jpg/full/!700,700/0/default.jpg

There are 23 images in there so it goes from:
32044138547039_01.jpg  --> 32044138547039_23.jpg

to get the manifest generated by canteloupe for each image it is:
https://iiif.itatti.harvard.edu/iiif/2/yashiro!32044138547039_01.jpg/info.json

**Comment by lklic** [comment]
I am still waiting on getting all of the letters.... this will take some time
